### ISSUE TITLE
make APP_NAME available as environment variable

### ISSUE THREAD

**Issue by psic4t** [jitsi/docker-jitsi-meet#1493]
APP_NAME should be available as an environment variable so that it can be used in docker-compose or k8s.

**Comment by saghul** [comment]
At the moment that is achievable with an interface-config-custom file.

**Comment by mdebus** [comment]
This is correct but we need to use bind mounts for this or to rebuild the container (for k8s). 

It would be a much better option to put this either in an environment variable or make it useable in BRANDING_DATA_URL (maybe the best way).

**Comment by psic4t** [comment]
bump

**Comment by saghul** [comment]
A PR would be welcome, we are unlikely to tackle this ourselves anytime soon.

**Comment by mircsicz** [comment]
Any Updates on that?

Just implemented a BRANDING_DATA_URL mod and looking forward to change APP_NAME

**Comment by saghul** [comment]
Feel free to make a PR and I'll review it.

**Comment by mircsicz** [comment]
Sorry, thats out of my area of excellence :-(
